hicommonwealth/commonwealth
# Mobile wallet signature confirmation modal

## Link to Issue
Closes: https://github.com/hicommonwealth/commonwealth/issues/6719

## Description of Changes
- Added a signature verification modal when logging-in with wallet connect on mobile devices

<img width="231" alt="image" src="https://github.com/hicommonwealth/commonwealth/assets/51641047/b89fbe26-3e6b-4371-9b99-ec450ac27519">


## "How We Fixed It"
N/A

## Test Plan
- Enable `FLAG_NEW_SIGN_IN_MODAL` in `.env`
- On your mobile device, visit the site, logout and go to any Ethereum based community or any non-community page 
- Attempt to login with wallet connect
- Verify you see the signature verification modal on mobile
- Verify that clicking on the `Sign with wallet` button opens your selected wallet for signature confirmation
- Verify that confirming the signature from wallet redirects back to app, logs you in, and closes AuthModal

## Deployment Plan
N/A


## Other Considerations
N/A
